var/spack/repos/builtin/packages/python/package.py

```diff
@@ -841,9 +846,9 @@ def python_include_dir(self):
         Returns:
             str: include files directory
         """
-        if 'python_inc' in self.config_vars:
+        try:
             return self.config_vars['python_inc']['false']
-        else:
+        except KeyError:
             return os.path.join('include', 'python{0}'.format(self.version.up_to(2)))
```

Comment:
I'm fine with these changes, but under what circumstances would `config_vars` have a `python_inc` key but not a `false` key?

Reply:
Good, both are initialized by the inline python script in the config_vars property method:
```py
608   config['python_inc'] = {}
609   config['python_lib'] = {}
```
So they might always be present, and if the output parsing wasn't successful, maybe there is nothing in it.

Anyways, I am able to reproduce it with the given packages on multiple machines with fresh setups I'm unlikely to be to only one who could trigger this in the future.